Dockerfile

```diff
@@ -6,19 +6,18 @@ COPY requirements.txt /tmp/
 
 RUN set -ex \
   && buildDeps=' \
-    build-essential \
+    gcc \
     libpq-dev \
-    python-dev \
 	' \
   && deps=' \
     gdal-bin \
-    git \
-    postgresql-client-9.4 \
+    gettext \
+    postgresql-client \
```

Comment:
Addressed in https://github.com/azavea/docker-django/pull/11/commits/b1fb0947361ebd165f87cfc5a572b643358f9747.